src/languages/es.ts

```diff
@@ -3461,4 +3461,7 @@ export default {
         offline:
             'Parece que estás desconectado. Desafortunadamente, Expensify Classic no funciona sin conexión, pero New Expensify sí. Si prefieres utilizar Expensify Classic, inténtalo de nuevo cuando tengas conexión a internet.',
     },
+    systemMessage: {
+        mergedWithCashTransaction: 'encontró un recibo para esta transacción',
```

Comment:
No period at the end like the english version?

```suggestion
        mergedWithCashTransaction: 'encontró un recibo para esta transacción.',
```